src/pattern-library/examples/select-next-non-popover-listbox.tsx

```diff
@@ -0,0 +1,70 @@
+import { useId, useState } from 'preact/hooks';
+
+import { SelectNext } from '../..';
+
+type ItemType = {
+  id: string;
+  name: string;
+  disabled?: boolean;
+};
+
+const items: ItemType[] = [
+  { id: '1', name: 'All students' },
+  { id: '2', name: 'Albert Banana' },
+  { id: '3', name: 'Bernard California' },
+  { id: '4', name: 'Cecelia Davenport' },
+  { id: '5', name: 'Doris Evanescence' },
+];
+
+export default function App() {
+  const [value, setValue] = useState<ItemType>();
+  const buttonId = useId();
+
+  return (
+    <div className="w-full">
+      <p>
+        When not using the <code>popover</code> API, the listbox will be
+        constrained by its container dimensions.
```

Comment:
I think "container dimensions" is ambiguous here. Perhaps "may be clipped by parent elements that are styled to hide overflow".